CHANGELOG.md

```diff
@@ -11,7 +11,11 @@ All notable changes to this project will be documented in this file.
 
 - **API:**
   - Added new endpoint to get agent stats from different components. ([#7128](https://github.com/wazuh/wazuh/issues/7128))
+  - Change Active Response endpoint function to create messages with new JSON format. ([#7312](https://github.com/wazuh/wazuh/pull/7312))
```

Comment:
Use past tense. Move to appropriate tag (Changed, not Added). Both entries (API and framework) are too similar, if there is no justification for the framework entry changes, remove it.

Reply:
corrected, sentence moved to `Changed` and removed the framework change entry